etc/config-template.toml

```diff
@@ -38,6 +38,18 @@ address = ""
 job = "tikv"
 
 [raftstore]
+# Base tick interval
```

Comment:
A log out could be helpful for info/debug. Yet it works only for normal users, not against system crackers.